pkg/controller/restore_controller.go

```diff
@@ -211,7 +278,9 @@ func (controller *restoreController) processRestore(key string) error {
 	logContext.Debug("Running processRestore")
 	ns, name, err := cache.SplitMetaNamespaceKey(key)
 	if err != nil {
-		return errors.Wrap(err, "error splitting queue key")
+		logContext.WithError(err).Error("unable to process restore: error splitting queue key")
```

Comment:
I think we have this returning an error in most/all of our existing controllers, we probably want to add an issue to go change that across the board

Reply:
+1